lib/redis_memo/memoize_query.rb

```diff
@@ -36,7 +35,9 @@ def memoize_table_column(*raw_columns, editable: true)
         end
       end
 
-      RedisMemo::MemoizeQuery.memoized_models[self.table_name] = self
```

Comment:
I guess I don't see this change yet in main or any open PRs: https://github.com/chanzuckerberg/redis-memo/pulls , so just from the diffs its not clear where the `memoized_models/enabled_models` var is used.

Reply:
It's used in this PR: https://github.com/chanzuckerberg/redis-memo/pull/4/files#diff-0ac6482c2039c84a625fc4a5c8a76be1f57cadcef1cd0739d92a8218dd316d22R58-R63